src/containers/login/Login.tsx

```diff
@@ -9,103 +8,112 @@ import { login } from '../../auth/ducks/thunks';
+    <div>
+      <Paragraph>
+        <Link to="/">FORGOT PASSWORD?</Link>
+      </Paragraph>
+
+      <Footer>
+        NEW TO SPEAK FOR THE TREES?
+        <br />
+        SIGN UP <Link to="/signup">HERE!</Link>
+      </Footer>
+    </div>
+  );
+
+  if (windowType === WindowTypes.Mobile) {
+    return (
+      <>
+        <Helmet>
+          <title>Login</title>
+          <meta name="description" content="Description goes here." />
+        </Helmet>
+        <LoginContainer>
+          <MobilePageHeader pageTitle="Log In" />
+          <LoginForm onFinish={onFinish} />
+          {ForgotPasswordFooter}
+        </LoginContainer>
+      </>
+    );
+  }
   return (
     <>
       <Helmet>
         <title>Login</title>
         <meta name="description" content="Description goes here." />
       </Helmet>
-      <div className="body-content-container">
+      <LoginContainer>
         <Row>
-          <Col span={10} className="input-container">
+          <InputContainer span={10}>
             <Title level={2} style={{ color: BLACK }}>
               Log In
             </Title>
-            <hr />
-            <Form
-              name="basic"
-              initialValues={{ remember: true }}
-              onFinish={onFinish}
-            >
-              <Row id="inputs">
-                <Col className="leftInput">
-                  <Form.Item
-                    name="email"
-                    rules={[
-                      {
-                        required: true,
-                        message: 'Please input your email!',
-                      },
-                    ]}
-                  >
-                    <Input placeholder="Email" />
-                  </Form.Item>
-                </Col>
-                <Col>
-                  <Form.Item
-                    name="password"
-                    rules={[
-                      {
-                        required: true,
-                        message: 'Please input your password!',
-                      },
-                    ]}
-                  >
-                    <Input.Password placeholder="Password" />
-                  </Form.Item>
-                </Col>
-              </Row>
+            <Line />
+            <LoginForm onFinish={onFinish} />
+            {ForgotPasswordFooter}
+          </InputContainer>
 
-              <Form.Item id={'loginButton'} style={{ marginBottom: '10px' }}>
-                <Button type="primary" htmlType="submit" size={'large'}>
-                  Log In
-                </Button>
-              </Form.Item>
-            </Form>
-
-            <Paragraph>
-              <Link to="/">Forgot Password?</Link>
-            </Paragraph>
-
-            <Footer>
-              NEW TO SPEAK FOR THE TREES?
-              <br />
-              SIGN UP <Link to="/signup">HERE!</Link>
-            </Footer>
-          </Col>
-          <Col span={2}></Col>
+          <Col span={`${width < 715 ? 1 : 2}`}></Col>
```

Comment:
Can we add a constant for this `715`? What does it represent?

Reply:
It's a breakpoint for the width - if `width < 715` an offset span of `2` is too large. Added constant `OffsetSpanBreakpoint`